rpcserver.go

```diff
@@ -3319,6 +3327,23 @@ func handleSendRawTransaction(s *rpcServer, cmd interface{}, closeChan <-chan st
 
 	// Use 0 for the tag to represent local node.
 	tx := btcutil.NewTx(&msgTx)
+
+	var totalSatoshiIn int64
+	inputs, err := fetchInputTxos(s, &msgTx)
+	for _, txIn := range inputs {
+		totalSatoshiIn += txIn.Value
+	}
+	var totalSatoshiOut int64
+	for _, txOut := range tx.MsgTx().TxOut {
+		totalSatoshiOut += txOut.Value
+	}
+	txFee := totalSatoshiIn - totalSatoshiOut
+	if txFee > int64(*c.MaxFeeRate) {
+		return nil, &btcjson.RPCError{
+			Code:    btcjson.ErrRPCDeserialization,
```

Comment:
doesn't seem like the correct response code, since the request was successfully deserialized

Reply:
returning `wire.RejectInvalid` now. I believe it is more appropriate